test/uk/gov/hmrc/merchandiseinbaggage/controllers/NavigatorSpec.scala

```diff
@@ -143,7 +143,7 @@ class NavigatorSpec extends DeclarationJourneyControllerSpec with PropertyBaseTa
         }
       }
 
-      s"from ${ReviewGoodsController.onPageLoad().url} navigates to ${GoodsDestinationController.onPageLoad()} " +
+      s"from ${ReviewGoodsController.onPageLoad().url} navigates to /declare-commercial-goods/goods-type-quantity/idx + 1 " +
```

Comment:
Why don't we use ${GoodsTypeController........} instead of an hard coded string?

Reply:
GoodsTypeController.onPageLoad() requires idx parameter. So I used String so I can just write idx + 1 … before it was wrongly using GoodsDestinationController, probs just a copy paste typo